packages/backend/index.js

```diff
@@ -60,20 +61,21 @@ app.get("/builders", async function (req, res) {
     .send(buildersSnapshot.docs.map((doc) => ({ id: doc.id, ...doc.data() })));
 });
 
-app.post("/", async function (request, response) {
+app.post("/sign", async function (request, response) {
   const ip =
     request.headers["x-forwarded-for"] || request.connection.remoteAddress;
   console.log("POST from ip address:", ip, request.body.message);
   if (
     request.body.message !==
     currentMessage.replace("**ADDRESS**", request.body.address)
   ) {
```

Comment:
This check looks unnecessary, as that mismatch would make the signature verification fail. For that to work we should send the `currentMessage` as first argument in the `ethers.utils.verifyMessage` call. Sending only data from the client to verify seems like a bad idea to me.

I've tested locally that verification fails if message mismatch and we verify with `currentMessage` instead of `req.body.message`.

We can remove this check making the code easier to follow and maintain. I can create an issue to tackle this if you agree with me.

Reply:
Will be tracked in #35. Resolving here